internal/uvm/create_lcow.go

```diff
@@ -76,7 +76,7 @@ func NewDefaultOptionsLCOW(id, owner string) *OptionsLCOW {
 		},
 		BootFilesPath:         filepath.Join(os.Getenv("ProgramFiles"), "Linux Containers"),
 		KernelFile:            "kernel",
-		KernelDirect:          false,
+		KernelDirect:          osversion.Get().Build >= 18286, // Use KernelDirect boot by default on all builds that support it.
```

Comment:
Minor nit, it may be worth putting a comment here to say that any caller must be manifested for this to work correctly. Just in case some third party comes along and wonders why it may not be working as expected.